src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -321,5 +323,11 @@ internal static DependencyGraphSpec CreateMinimalDependencyGraphSpec(string proj
 
             return dgSpec;
         }
+
+        public Task RegisterRestoreInfoSourceAsync(IVsProjectRestoreInfoSource restoreInfoSource, CancellationToken cancellationToken)
+        {
+            return Task.CompletedTask;
+            // TODO: https://github.com/NuGet/Home/issues/10678
```

Comment:
Are you going to remove this TODO ?

Reply:
The TODO is there to indicate that it's happening later.